test/helpers/start-app.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable node/no-unpublished-require */
+/* eslint-disable no-console */
+
+const { spawn } = require('child_process');
+const execa = require('execa');
+const path = require('path');
+
+// resolved from the root of the project
+const classicAppDir = path.resolve('./test/fixtures/classic-app');
+const execOpts = { cwd: classicAppDir, stderr: 'inherit' };
+module.exports = async function startApp() {
+  console.log('installing deps');
+
+  await execa('rm', ['-rf', 'node_modules'], execOpts);
+  await execa('yarn', ['install'], execOpts);
+
+  console.log('starting serve');
+
+  // We use spawn for this one so we can kill it later without throwing an error
+  const emberServe = spawn('yarn', ['start'], execOpts);
```

Comment:
This one was tough because of the comment.  `execa` resolves when the process completes.  Therefore if you return it and `await` on it you get stuck.  To fix this I had to wrap `emberServe` in a POJO so that I could kill it later.